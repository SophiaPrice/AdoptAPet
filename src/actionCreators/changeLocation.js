import { SET_LOCATION } from "../actions";

export default function changeLocation(location) {
  return {
    type: SET_LOCATION,
    payload: location
  };
}
