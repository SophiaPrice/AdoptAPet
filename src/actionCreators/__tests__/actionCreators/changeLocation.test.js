import changeLocation from "../../changeLocation";

test("changeLocation snapshot", () => {
  expect(changeLocation("Seattle, WA")).toMatchSnapshot();
});
