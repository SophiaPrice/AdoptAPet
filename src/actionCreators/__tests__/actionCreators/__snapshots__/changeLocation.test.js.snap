// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changeLocation snapshot 1`] = `
Object {
  "payload": "Seattle, WA",
  "type": "SET_LOCATION",
}
`;
